src/pages/core/guides/javascript/how-to-create-a-core-nft-asset.md

```diff
@@ -0,0 +1,371 @@
+---
+title: How to Create a Core NFT Asset
+metaTitle: How to Create a Core NFT Asset
+description: Learn how to create a Core NFT Asset on the Solana blockchain with the Metaplex Core package.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-18-2024'
+---
+
+This is an intial guide on how to create a next-gen Digital Asset Nft (Core Asset) on the Solana blockchain using the Metaplex Mpl Core protocol. Mpl Core Assets provide the next wave of NFT projects on Solana with greater creativity and a broad dynamic experiance for both creators and owners with Mpl Core's simplied design and unique plugin system.
```

Comment:
Nit: I still think "Mpl Core" should be called "MPL Core" everywhere but can live with it as "Mpl Core"